### ISSUE TITLE
Expand button is missing tooltip

### ISSUE THREAD

**Issue by benmccann** [photoprism/photoprism#823]
The expand button in the lefthand menu doesn't have a tooltip / alt text like the icons below it.

I was also going to suggest that the lefthand menu be expanded by default. I was having a really hard time with just the icons and it took me quite awhile to realize that the menu was expandable

**Comment by lastzero** [comment]
We'll add yet another config option for this.

**Comment by graciousgrey** [comment]
Good catch, I will add the tooltip.
Regarding the the default state of the menu, we have a related ticket: https://github.com/photoprism/photoprism/issues/378

**Comment by benmccann** [comment]
Confirmed that I see the tooltip now. Thanks!

**Comment by lastzero** [comment]
Thank you :+1:
